vm/src/version.rs

```diff
@@ -4,10 +4,10 @@
 use chrono::{prelude::DateTime, Local};
 use std::time::{Duration, UNIX_EPOCH};
 
-// = 3.12.0alpha
+// = 3.13.1alpha
 pub const MAJOR: usize = 3;
-pub const MINOR: usize = 12;
-pub const MICRO: usize = 0;
+pub const MINOR: usize = 13;
+pub const MICRO: usize = 1;
```

Comment:
```suggestion
pub const MICRO: usize = 0;
```
we usually keep this zero because we always have release level alpha. This number doesn't affect python libraries' behavior. So 0 is always fine.